runtimes/react-native/loading-rive-files.mdx

```diff
@@ -0,0 +1,328 @@
+---
+title: "Loading Rive Files"
+description: "How to use Rive files with the Rive React Native runtime. "
+---
+
+<Tabs>
+  <Tab title="New Runtime (Recommended)">
+    There are several ways to load Rive files in your React Native projects using the new runtime:
+
+    - **Option 1: Using `require()`** - Load files from your project directory (recommended for development and OTA updates)
+    - **Option 2: URL** - Load files from a remote URL
+    - **Option 3: Resource name** - Load files from native asset bundles
+    - **Option 4: ArrayBuffer** - Load files from binary data
+
+    All loading methods use the `useRiveFile` hook, which returns a `RiveFile` object that you pass to the `RiveView` component via the `file` prop.
+
+### Option 1: Using `require()` (Recommended)
+
+    Loading Rive files using `require()` is recommended because it doesn't require a native rebuild when you update the file. With Expo, this also enables Over The Air (OTA) updates.
```

Comment:
```suggestion
    Loading Rive files using `require()` is recommended because it doesn't require a native rebuild when you update the Rive file. With Expo, this also enables Over The Air (OTA) updates.
```